auth/src/api/handlers.rs

```diff
@@ -69,56 +68,22 @@ pub(crate) async fn update_user_tier(
 }
 
 pub(crate) async fn put_user_reset_key(
-    session: ReadableSession,
     State(user_manager): State<UserManagerState>,
     key: Option<Key>,
 ) -> Result<(), Error> {
-    let account_name = match session.get::<String>("account_name") {
-        Some(account_name) => account_name.into(),
-        None => match key {
-            Some(key) => user_manager.get_user_by_key(key.into()).await?.name,
-            None => return Err(Error::Unauthorized),
-        },
+    let account_name = match key {
+        Some(key) => user_manager.get_user_by_key(key.into()).await?.name,
+        None => return Err(Error::Unauthorized),
     };
```

Comment:
We should be able to switch the parameter to be just `key: Key`, since it's a mandatory argument now. We do the same in `convert_key`.